### ISSUE TITLE
Many `sqlPlpgsqlKeyword` keywords are not properly highlighted

### ISSUE THREAD

**Issue by LaurentValade** [lifepillar/pgsql.vim#25]
Firstly, thank you for your package.

I just installed it on Debian (**Debian GNU/Linux 10 (buster)** et **VIM - Vi IMproved 8.1 (2018 May 18, compiled Jun 15 2019 16:41:15)**) but `sqlPlpgsqlKeyword` keywords are not properly highlighted as shown by this screenshot of a dummy SQL file :
<img width="453" alt="screenshot-sans-bidouille" src="https://user-images.githubusercontent.com/56821524/115389391-515fa880-a1dd-11eb-9bf1-28ed4d7215b8.png">

As a temporary (very ugly) fix, I simply add all those keywords to `sqlKeyword` with :
```vim
syn keyword sqlKeyword contained alias all array as begin by case close collate column constant
syn keyword sqlKeyword contained constraint continue current current cursor datatype declare
syn keyword sqlKeyword contained detail diagnostics else elsif end errcode exception execute
syn keyword sqlKeyword contained exit fetch for foreach forward found from get hint if
syn keyword sqlKeyword contained into last loop message move next no notice open perform prepare
syn keyword sqlKeyword contained query raise relative return reverse rowtype schema
syn keyword sqlKeyword contained scroll slice sqlstate stacked strict table tg_argv tg_event
syn keyword sqlKeyword contained tg_level tg_name tg_nargs tg_op tg_relid tg_relname
syn keyword sqlKeyword contained tg_table_name tg_table_schema tg_tag tg_when then type using
syn keyword sqlKeyword contained while
```
in `~/.vim/pack/plugins/start/pgsql/syntax/pgsql.vim` and I got that (which is enough for me for the moment!) :
<img width="452" alt="screenshot-avec-bidouille" src="https://user-images.githubusercontent.com/56821524/115389637-9be12500-a1dd-11eb-9803-41d548072e1f.png">

**Comment by lifepillar** [comment]
PL/pgSQL keywords are highlighted only inside PL/pgSQL functions. Try:

```sql
create or replace function Foo() returns void
language plpgsql as
$$
begin
  -- Write keywords here.
end;
$$;
```

Is there a reason why you need highlighting outside functions?

**Comment by LaurentValade** [comment]
You are completely right, there is no need of PL/pgSQL syntax highlighting outside PL/pgSQL functions.  
I don’t know where I screwed up, but when I created this ticket, PL/pgSQL syntax highlighting was broken inside functions.  
Now, it works fine (as soon as `$$` is present, PL/pgSQL sh is activated), sorry about bad alert!
<img width="256" alt="screenshot-sans-bidouille ok 0 complete" src="https://user-images.githubusercontent.com/56821524/115580537-76c8e100-a2c7-11eb-9794-6e1a62c5f1c3.png"> <img width="256" alt="screenshot-sans-bidouille ok 1 two-dollars" src="https://user-images.githubusercontent.com/56821524/115580554-7a5c6800-a2c7-11eb-970e-b55e37575034.png"> <img width="256" alt="screenshot-sans-bidouille ok 2 only-one-dollar" src="https://user-images.githubusercontent.com/56821524/115580566-7d575880-a2c7-11eb-9d51-89bfa8eb5111.png">
